src/_gettsim/parameters/elterngeld.yaml

```diff
@@ -90,7 +105,7 @@ eink_schritt_korrektur:
   2007-01-01:
     scalar: 2
     reference: Art. 1 G. v. 05.12.2006 BGBl. I S. 2748
```

Comment:
Shouldn't it be Art. 2 Absatz 2?

Reply:
Artikel 1 is the "Gesetz zur Änderung des BEEG", §2 Abs. 2 is part of that.